README.md

```diff
@@ -4,16 +4,20 @@ A GitHub action that runs [black code formatter](https://github.com/ambv/black)
 
 ## Example Workflow
 
-```workflow
-workflow "Example Workflow" {
-  on = "push"
-  resolves = ["Lint"]
-}
+```yaml
+name: Code Quality Checks
 
-action "Lint" {
-  uses = "lgeiger/black-action@master"
-  args = ". --check"
-}
+on: [push]
+
+jobs:
+  black:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
I think this can be set to `actions/checkout@v2`

Reply:
+1, but I think it's more beneficial for the users of this action to get this PR merged as it is than to wait for the author to change this single character first...